src/builtin.d

```diff
@@ -48,35 +49,35 @@ extern (C++) Expression eval_sin(Loc loc, FuncDeclaration fd, Expressions* argum
 {
     Expression arg0 = (*arguments)[0];
     assert(arg0.op == TOKfloat64);
-    return new RealExp(loc, sinl(arg0.toReal()), arg0.type);
+    return new RealExp(loc, TargetReal.sin(arg0.toReal()), arg0.type);
```

Comment:
>>  are these required built-ins available for the type you use in GDC?

Of course they aren't!  Built-ins aren't handled explicitly in GDC, instead a tree is generated and passed to const-fold in the middle-end to see if it returns a literal.